spot_description/urdf/spot_arm_macro.urdf

```diff
@@ -0,0 +1,223 @@
+<?xml version="1.0"?>
+<robot name="spot_arm" xmlns:xacro="http://www.ros.org/wiki/xacro">
+  <xacro:macro name="load_arm" params="
+    tf_prefix" >
+
+    <link name="${tf_prefix}arm_link_sh0">
+      <visual>
+        <origin xyz="0 0 0" rpy="0 0 0"/>
+        <geometry>
+          <mesh filename="package://spot_description/meshes/arm/visual/arm_link_sh0.obj"/>
+        </geometry>
+        <material name="arm_link_sh0"> <color rgba="0.2 0.2 0.2 1"/></material>  
+      </visual>
+      <collision>
+        <geometry>
+          <mesh filename="package://spot_description/meshes/arm/collision/arm_link_sh0.stl" />
+        </geometry>
+      </collision>
+      <inertial>
+        <origin rpy="0.0013490340912847287 -0.15433618253071657 0.0066354181456347305" xyz="-0.012372709810733795 -4.5211403630673885e-05 -0.02103196457028389"/>
+        <mass value="2.3364000000000003"/>
+        <inertia ixx="0.009014303010689815" ixy="0.0" ixz="0.0" iyy="0.005835102417829581" iyz="0.0" izz="0.00778747846359954"/>
+      </inertial>
+    </link>
+    <joint name="${tf_prefix}arm_sh0" type="revolute">
```

Comment:
What do you think about adding a `/` after ${tf_prefix}? Should the user be responsible for that?